pkg/controller/backup_sync_controller.go

```diff
@@ -37,6 +38,11 @@ import (
 	"github.com/vmware-tanzu/velero/pkg/plugin/clientmgmt"
 )
 
+const (
+	backupSyncPeriodKey = "backupSyncPeriod"
+	skipBackupSyncKey   = "skipBackupSync"
```

Comment:
Would it make sense to use `backupSyncPeriod: 0` be a sentinel value for skipping sync, so we only have one variable to check?

Reply:
Yes, that makes sense. I had initially thought of doing it that way but figured adding a config variable for skipping sync would make it more explicit. I'll update the PR.

Reply:
That's certainly true that it's more explicit, which tends to be good. Curious what others think, though.

Reply:
I have the changes to use backupSyncPeriod: 0 as a sentinel value for skipping sync all ready, just waiting for input from other reviewers then will update the PR.